cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Is this file supposed to be commited?

Reply:
@martina-if I removed it and the build failed. The testing framework seems to expect it to be there, even if it doesn't do anything.

BTW I need a new review now please. 😀